jekyll/troubleshooting.markdown

```diff
@@ -29,15 +29,15 @@ As an example, the activation script for `zsh` using `rbenv` as a version manage
 ```
 
 After activating the Ruby version, we then proceed to boot the server gem (`ruby-lsp`). To avoid having users include
-the `ruby-lsp` in their `Gemfile`, we currently create a custom bundle under the `.ruby-lsp` directory inside your
+the `ruby-lsp` in their `Gemfile`, we currently create a composed bundle under the `.ruby-lsp` directory inside your
```

Comment:
We should probably link to the composed bundle page from here instead of explaining everything again.